app/schema/answer_factory.py

```diff
@@ -0,0 +1,34 @@
+from app.schema.questionnaire import QuestionnaireException
+
+from app.schema.answers.checkbox_answer import CheckboxAnswer
+from app.schema.answers.currency_answer import CurrencyAnswer
+from app.schema.answers.date_answer import DateAnswer
+from app.schema.answers.integer_answer import IntegerAnswer
+from app.schema.answers.percentage_answer import PercentageAnswer
+from app.schema.answers.positiveinteger_answer import PositiveIntegerAnswer
+from app.schema.answers.radio_answer import RadioAnswer
+from app.schema.answers.textarea_answer import TextareaAnswer
+from app.schema.answers.textfield_answer import TextfieldAnswer
+
+
+class AnswerFactory(object):
+    @staticmethod
```

Comment:
There's a generic factory in (https://github.com/ONSdigital/eq-survey-runner/blob/master/app/utilities/factory.py) that does this, you can just create one of them here and register the types e.g

AnswerFactory = factory.Factory()
AnswerFactory.register_all(...)